lib/slide_hub/convert_util.rb

```diff
@@ -87,5 +95,21 @@ def get_local_file_list(dir, extension)
       end
       list.sort
     end
+
+    private
+
+      def exec_command(cmd)
+        logger.info(cmd)
+        Open3.popen3(cmd) do |_i, o, e, w|
+          out = o.read
+          err = e.read
+          logger.info out if out.length > 0
+          logger.error err if err.length > 0
+          return w.value.exitstatus.zero?
+        end
+      rescue StandardError => e
```

Comment:
__[Reek]__

[UncommunicativeVariableName] SlideHub::ConvertUtil#exec_command has the variable name 'e'


<!-- Comment by [Sider](https://sider.review) -->